datafusion/physical-plan/src/aggregates/mod.rs

```diff
@@ -1071,13 +1072,26 @@ fn get_aggregate_expr_req(
     aggr_expr: &AggregateFunctionExpr,
     group_by: &PhysicalGroupBy,
     agg_mode: &AggregateMode,
+    include_soft_requirement: bool,
```

Comment:
can we please update the documentation of this function to include a description of what `include_soft_requirement` means?